avocado/utils/asset.py

```diff
@@ -40,6 +40,12 @@
 log = logging.getLogger('avocado.test')
 
 
+class UnsupportProtocolError(EnvironmentError):
```

Comment:
Wouldn't `UnsupportedProtocolError` sound better? Apart from that yes, it's an improvement.

Reply:
Actually do you think people will tend to use this frequently? I'd probably just go with `EnvironmentError` along with custom message.

Reply:
I'm always a fan of explicit, but the naming yes, *Unsupported* is probably what I meant and failed to notice that wasn't written properly.